wsagent/che-core-api-project-shared/src/main/java/org/eclipse/che/api/project/shared/dto/event/GitBranchCheckoutEventDto.java

```diff
@@ -0,0 +1,30 @@
+/*******************************************************************************
+ * Copyright (c) 2012-2016 Codenvy, S.A.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *   Codenvy, S.A. - initial API and implementation
+ *******************************************************************************/
+package org.eclipse.che.api.project.shared.dto.event;
+
+import com.google.common.annotations.Beta;
+
+import org.eclipse.che.dto.shared.DTO;
+
+/**
+ * To transfer branch name after git checkout operation
```

Comment:
Yes, that is exactly what they are saying [here]( https://git-scm.com/docs/git-checkout). And yes, it is an example, it is not really intended to be production ready. You are right twice in a row! But anyway if you could pay a little attention to the name of the interface you would notice that it is named `GitBranchCheckoutEventDto`, I'm pointing to words **branch** and **checkout**. Thus it seems obvious for me that DTO is related to branch checkout events only. Still I don't mind if you add a more explanatory description.